### PR TITLE
Download java sources lazily

### PR DESCRIPTION
**Pull request by snjeza**
Requires https://github.com/eclipse/eclipse.jdt.ls/pull/986

Signed-off-by: Snjezana Peco <snjezana.peco@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,9 @@ The following settings are supported:
 *New in 0.42.0:*
 * `java.foldingRange.enabled`: Enable/disable smart folding range support. If disabled, it will use the default indentation-based folding range provided by VS Code.
 
+*New in 0.43.0:*
+* `java.maven.downloadSources`: Enable/disable downloading Maven source artifacts.
```
Comment (fbricon) [inline README.md]: ... on startup

File: package.json [hunk 2]
```diff
@@ -131,6 +131,12 @@
           "description": "Enable/disable the Maven importer.",
           "scope": "window"
         },
+        "java.maven.downloadSources": {
+          "type": "boolean",
+          "default": false,
+          "description": "Enable/disable downloading Maven source artifacts.",
```
Comment (fbricon) [inline package.json]: Enable/disable eager download of Maven source artifacts

**Comment by snjeza** [comment]
@fbricon I have updated the PR.

File: README.md [hunk 3]
```diff
@@ -103,6 +103,9 @@ The following settings are supported:
 *New in 0.42.0:*
 * `java.foldingRange.enabled`: Enable/disable smart folding range support. If disabled, it will use the default indentation-based folding range provided by VS Code.
 
+*New in 0.43.0:*
+* `java.maven.downloadSources`: Enable/disable downloading Maven source artifacts on startup.
```
Comment (fbricon) [inline README.md]: `Enable/disable eager download of Maven source artifacts`

sorry, I changed my mind earlier and you missed my comment update

**Pull request merged**